jwst/extract_1d/extract.py

```diff
@@ -338,50 +338,45 @@ def get_extract_parameters(
                     # Note: extract_params['dispaxis'] is not assigned. This is done later, possibly slit by slit.
 
                     if meta.target.source_type == "EXTENDED":
-                        log.info("Target is extended, so the entire region will be extracted.")
                         shape = input_model.data.shape
-                        extract_params['src_coeff'] = None
-                        extract_params['bkg_coeff'] = None
-                        extract_params['bkg_fit'] = None
-                        extract_params['bkg_order'] = 0
-                        extract_params['subtract_background'] = False
-                        extract_params['use_source_posn'] = False
```

Comment:
So does the removal of this setting mean that we will always try to use the computed source position to set the center of the extraction region for extended sources? I'm not sure that's a good idea. That could end up overriding the start/stop limits entered by the user, or even if the user doesn't give any it could also end up trying to shift the default limits (the whole region) later on in the processing. Although there might be guard rails in place to make sure the shifted start/stop values don't go beyond the limits of the data array.

Reply:
After a regression test run, it appears I did a *bad thing*. Looking into it.